src/vs/platform/telemetry/node/workbenchCommonProperties.ts

```diff
@@ -29,6 +29,23 @@ export async function resolveWorkbenchCommonProperties(storageService: IStorageS
 	result['common.isNewSession'] = !lastSessionDate ? '1' : '0';
 	// __GDPR__COMMON__ "common.instanceId" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
 	result['common.instanceId'] = instanceId;
+	// __GDPR__COMMON__ "common.remoteAuthority" : { "classification": "SystemMetaData", "purpose": "PerformanceAndHealth" }
+	result['common.remoteAuthority'] = cleanRemoteAuthority(remoteAuthority);
 
 	return result;
 }
+
+function cleanRemoteAuthority(remoteAuthority?: string): string {
+	if (!remoteAuthority) {
+		return 'none';
+	}
+
+	// Whitelisted remote authorities
+	['ssh-remote', 'dev-container', 'wsl'].forEach((res: string): string | void => {
+		if (remoteAuthority!.indexOf(res) === 0) {
+			return res;
```

Comment:
You can't return out of a foreach like that

Reply:
👍

Reply:
stupid question here, should we validate `ssh-remote+` instead? Not sure if someone will use authority like `ssh-remote-something+blabla`.